pkg/templates/loader/terraform_loader.go

```diff
@@ -66,6 +66,10 @@ func (l *TerraformLoader) Load(
 
 // loadMod load the terraform module.
 func (l *TerraformLoader) loadMod(rootDir string) (*tfconfig.Module, error) {
+	if !tfconfig.IsModuleDir(rootDir) {
+		return nil, fmt.Errorf("no terraform configuration files found")
```

Comment:
How do we handle the error at the end, should it be an error?

Reply:
For catalog, walrus server will log these errors and skip this version, continually importing other versions, no error in the template status; for imported template with URL, will show this error in the template status.